src/Avalonia.Controls/TextBox.cs

```diff
@@ -443,6 +449,15 @@ public char PasswordChar
             set => SetValue(CaretBrushProperty, value);
         }
 
+        /// <summary>
+        /// Gets or sets a brush that is used for the text caret
```

Comment:
Wrong comments (you copy/pasted), missing period.

Reply:
Fixed